ruby/vampires.rb

```diff
@@ -0,0 +1,147 @@
+
+
+application_number = 0
+puts "How many employees will you be processing?"
+
+#Use this while loop to ensure user inputs a valid number. 
+while true
+     employees_to_process = gets.chomp.to_i
+          if employees_to_process > 0 && employees_to_process < 1000000
+            break
+          else 
+            puts "Please answer with a number between 0 and 1000000"
+          end    
+      end 
+
+#This until loop will repeat program until all employees have been processed. 
+until application_number == employees_to_process
+
+#VAMPIRE DETECTION QUESTIONS 
+
+#Set Boolean to inspect whether the name entered matches Tu Fang or Drake Cula. 
+    puts "What the applicant's name?"
+    name = gets.chomp.downcase
+        if name == "drake cula" || name == "tu fang"
+          name = false
+#Setting a condition that sets either name as "Definitely a Vampire" here would stop the survey questions. 
+#I dont set that condition until all questions have been asked.   
+        else 
+          name = true
+        end       
+
+    puts "How old are you?"
+#Use this while loop to ensure user inputs a valid number.     
+      while true
+      age_given = gets.chomp.to_i
```

Comment:
this integer error handling is not necessary.  google the ruby docs for how to check if a data type is a valid integer in ruby